examples/29_rgb_depth_aligned.py

```diff
@@ -0,0 +1,86 @@
+# Otherwise (False), the aligned depth is automatically upscaled to 1080p
+downscaleColor = True
+
+pipeline = dai.Pipeline()
+
+cam = pipeline.createColorCamera()
+cam.setBoardSocket(dai.CameraBoardSocket.RGB)
+cam.setResolution(dai.ColorCameraProperties.SensorResolution.THE_1080_P)
+if downscaleColor: cam.setIspScale(2, 3)
+# For now, RGB needs fixed focus to properly align with depth.
+# This value was used during calibration
+cam.initialControl.setManualFocus(130)
+
+rgbOut = pipeline.createXLinkOut()
+rgbOut.setStreamName("rgb")
+cam.isp.link(rgbOut.input)
+
+left = pipeline.createMonoCamera()
+left.setResolution(dai.MonoCameraProperties.SensorResolution.THE_720_P)
+left.setBoardSocket(dai.CameraBoardSocket.LEFT)
+
+right = pipeline.createMonoCamera()
+right.setResolution(dai.MonoCameraProperties.SensorResolution.THE_720_P)
+right.setBoardSocket(dai.CameraBoardSocket.RIGHT)
+
+stereo = pipeline.createStereoDepth()
+stereo.setConfidenceThreshold(200)
+# LR-check is required for depth alignment
+stereo.setLeftRightCheck(True)
+stereo.setDepthAlign(dai.CameraBoardSocket.RGB)
+left.out.link(stereo.left)
+right.out.link(stereo.right)
+
+depthOut = pipeline.createXLinkOut()
+depthOut.setStreamName("depth")
+# Currently we use the 'disparity' output. TODO 'depth'
+stereo.disparity.link(depthOut.input)
+
+# Pipeline defined, now the device is connected to
+with dai.Device(pipeline) as device:
+    # Start pipeline
+    device.startPipeline()
+
+    qColor = device.getOutputQueue(name="rgb",   maxSize=4, blocking=False)
+    qDepth = device.getOutputQueue(name="depth", maxSize=4, blocking=False)
+
+    frameRgb = None
+    frameDepth = None
+
+    while True:
+        queueName = device.getQueueEvent(("rgb", "depth"))
+
+        packet = device.getOutputQueue(queueName).get()
+
+        if queueName == "rgb":
+            frameRgb = packet.getCvFrame()
+            cv2.imshow("rgb", frameRgb)
+        elif queueName == "depth":
+            frameDepth = packet.getFrame()
+            # Optional, extend range 0..95 -> 0..255, for a better visualisation
+            if 1: frameDepth = (frameDepth * 255. / 95).astype(np.uint8)
+            # Optional, apply false colorization
+            if 1: frameDepth = cv2.applyColorMap(frameDepth, cv2.COLORMAP_HOT)
+            frameDepth = np.ascontiguousarray(frameDepth)
+            cv2.imshow("depth", frameDepth)
+
+        # Blend when both received
+        if frameRgb is not None and frameDepth is not None:
+            # Need to have both frames in BGR format before blending
+            if len(frameDepth.shape) < 3:
+                frameDepth = cv2.cvtColor(frameDepth, cv2.COLOR_GRAY2BGR)
+            # TODO add a slider to adjust blending ratio
+            blended = cv2.addWeighted(frameRgb, 0.6, frameDepth, 0.4 ,0)
+            cv2.imshow("rgb-depth", blended)
+            frameRgb = None
+            frameDepth = None
+
+        if cv2.waitKey(1) == ord('q'):
+            break
```

Comment:
With above changes in place, you can do just `exit()` here, and it should exit cleanly